### PR TITLE
8225122: Test AncestorResized.java fails when Windows desktop is scaled.

### PR DESCRIPTION
**Pull request by mrserb**
This test depends on the next windows UI performance option:
“Show window contents while dragging”, see https://answers.microsoft.com/en-us/windows/forum/all/in-windows-7-how-do-i-show-window-contents-while/18e3aee2-bcc7-4005-a45d-c5613cf6eb3c

In the default configuration, the windows may decide that the system is slow and this option should be disabled. I have found that this usually happens in the virtual environment.

When that option is disabled the resize events have come when the user completes the resize, before that the only border "rectangle" is moved.

This fix changes the test to release the mouse and complete the resize, and only after that checks the result.

Also, some old machinery is removed.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8225122](https://bugs.openjdk.java.net/browse/JDK-8225122): Test AncestorResized.java fails when Windows desktop is scaled.

### Reviewers
 * [Alexander Zuev](https://openjdk.java.net/census#kizune) (@azuev-java - **Reviewer**) ⚠️ Review applies to 7f5a6c24c27e0bfa70794619f6a49a3344167ce4
 * [Alexey Ivanov](https://openjdk.java.net/census#aivanov) (@aivanov-jdk - **Reviewer**) ⚠️ Review applies to 7f5a6c24c27e0bfa70794619f6a49a3344167ce4

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/5711/head:pull/5711` \
`$ git checkout pull/5711`

Update a local copy of the PR: \
`$ git checkout pull/5711` \
`$ git pull https://git.openjdk.java.net/jdk pull/5711/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 5711`

View PR using the GUI difftool: \
`$ git pr show -t 5711`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/5711.diff">https://git.openjdk.java.net/jdk/pull/5711.diff</a>

</details>

### CODE REVIEW

File: test/jdk/java/awt/event/HierarchyEvent/AncestorResized/AncestorResized.java [hunk 1]
```diff
@@ -112,177 +104,12 @@ private static void init()
         // ... and start resizing
         robot.mousePress( InputEvent.BUTTON1_MASK );
         robot.mouseMove(bounds.x + bounds.width + 20, bounds.y + bounds.height + 15);
-        Util.waitForIdle(robot);
+        robot.mouseRelease(InputEvent.BUTTON1_MASK);
```
Comment (mrserb) [inline test/jdk/java/awt/event/HierarchyEvent/AncestorResized/AncestorResized.java:107]: This is the actual change, mouseRelease is moved from the end of the test to the line above.

**Comment by mrserb** [comment]
Any volunteers?

**Comment by mrserb** [comment]
Any volunteers?

File: test/jdk/java/awt/event/HierarchyEvent/AncestorResized/AncestorResized.java [hunk 2]
```diff
@@ -61,8 +54,7 @@ public void ancestorResized(HierarchyEvent ce) {
         }
     }
 
-    private static void init()
-    {
+    public static void main(String[] args) throws Exception {
         Frame frame;
```
Comment (azuev-java) [inline test/jdk/java/awt/event/HierarchyEvent/AncestorResized/AncestorResized.java:58]: Not a big fan of the split declaration and initialization of all the variables but that existed before the fix and it is purely cosmetic. Other than that looks good.

**Review by azuev-java — APPROVED** [review]

**Pull request closed without merging**